composer.json

```diff
@@ -13,7 +13,13 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5",
-        "squizlabs/php_codesniffer": "^3.0"
+        "squizlabs/php_codesniffer": "^3.0",
+        "silverstripe/subsites": "^2",
+        "silverstripe/taxonomy": "^2",
+        "silverstripe/contentreview": "^4"
+    },
+    "conflict": {
+        "silverstripe/reports": "<4.11.0"
```

Comment:
Why is this conflict added?

Reply:
If this is because of the parameter in `count()`, then we should actually conflict with ``>=4.11.0` on this branch without making the change here, since that parameter was added in 4.11 but this is the branch that corresponds with 4.10

Reply:
Still need to remove the actual change to the count method.

Reply:
updated